lib/action.js

```diff
@@ -245,6 +245,28 @@ function action(system) {
 
 	});
 
+	// If a user wants to abort a single button actions
+	self.system.on('action_abort_one', function(bank) {
```

Comment:
instead of having to use bank[0] and bank[1] it would be nicer to have two parameters, page and bank.

I would prefer a name that refers to 'bank' rather than 'one' (eg `action_delayed_abort_bank`), but that will require a change in the module using this too

Reply:
This name and parameter is defined in the bitfocus-companion module already, though that could be changed to suit.  It appears the ["Abort actions on button"](https://github.com/bitfocus/companion-module-bitfocus-companion/blob/7a84df70064e8a3907ecbff5ea66359885d4a58b/index.js#L717) action in internal has existed for a while without code to execute having been implemented.  Though I agree that `(page, bank)` as the parameters follows conventions elsewhere in the software.

Reply:
The action 'request' in the internal module came about while adding an option for 'this button' on button modifiers. The actual execution was in this module. At the moment it is a do-nothing action since the listener is in this patch.

Reply:
The only module using this is the `bitfocus-companion` internal module.